src/views/App/index.tsx

```diff
@@ -22,6 +22,7 @@ import Sandbox from 'views/Sandbox';
 import SystemIntake from 'views/SystemIntake';
 import TermsAndConditions from 'views/TermsAndConditions';
 import TimeOutWrapper from 'views/TimeOutWrapper';
+import UserInfoWrapper from 'views/UserInfoWrapper';
```

Comment:
Okta gets finicky when we use the tokenManager too many places in the same file, so I spun up a new, small, wrapper to handle storing user information. It would potentially be nice to store this on login, but I think we would lose the context if the user refreshed the page.